bigtable/google/cloud/bigtable/batcher.py

```diff
@@ -143,8 +143,15 @@ def mutate_rows(self, rows):
             self.mutate(row)
 
     def flush(self):
-        """ Sends the current. batch to Cloud Bigtable. """
-        if len(self.rows) is not 0:
+        """ Sends the current. batch to Cloud Bigtable.
+        For example:
+
+        .. literalinclude:: snippets.py
+            :start-after: [START bigtable_batcher_flush]
+            :end-before: [END bigtable_batcher_flush]
+
+        """
+        if len(self.rows) != 0:
```

Comment:
This change improves on the former (which could even be a bug), but would be clearer as just `if self.rows:`.

Reply:
Definintely. Done.